framework_tvb/tvb/adapters/uploaders/mat_timeseries_importer.py

```diff
@@ -145,7 +145,7 @@ def create_region_ts(self, data_shape, connectivity):
                                   % (data_shape[1], connectivity.number_of_regions))
         ts_idx = TimeSeriesRegionIndex()
         ts_idx.fk_connectivity_gid = connectivity.gid
-        ts_idx.has_surface_mapping = True
+        ts_idx.has_surface_mapping = True if ts_idx.fk_region_mapping_gid is not None else False
```

Comment:
This will always be False since you never set the ts_idx.fk_region_mapping_gid in this importer